src/main/java/seedu/address/commons/util/DateUtil.java

```diff
@@ -61,16 +63,29 @@ public static String decodeDate(LocalDate date) {
 
     /**
      * Decodes a date passed as a LocalDate into a String in the EEEE, dd MMM uuuu format.
+     *
      * @param date A LocalDate object.
      * @return A date String in the EEEE, dd MMM uuuu format.
      */
     public static String decodeDateWithDay(LocalDate date) {
         requireNonNull(date);
-        return date.format(DateTimeFormatter.ofPattern("EEEE, dd MMM uuuu"));
+        return date.format(DateTimeFormatter.ofPattern("EEE, dd MMM uuuu"));
+    }
+
+    /**
+     * Decodes a date passed as a LocalDate into a String in the EEEE format.
+     *
+     * @param date A LocalDate object.
+     * @return A date String in the EEEE format.
+     */
+    public static String decodeDateIntoDay(LocalDate date) {
+        requireNonNull(date);
+        return date.format(DateTimeFormatter.ofPattern("EEEE"));
```

Comment:
Was day here intentionally left as 4 chars? as it has been shortened to 3 chars above.

Reply:
Yes, i've made more space for the day label. Tried both, i decided this looks nicer.

Reply:
👍